src/main/java/com/ibm/eventstreams/kafkaconnect/plugins/xml/engines/StructToXmlBytes.java

```diff
@@ -140,20 +137,17 @@ private void processSchema(Document doc, Element sequenceElement, String name, S
                     if (schema.valueSchema().isOptional()) {
                         schemaElement.setAttribute("minOccurs", "0");
                     }
-                }
-                else {
+                } else {
                     final Element arrayTypeDef = addNode(doc, schemaElement, "xs:complexType");
                     final Element arraySeq = addNode(doc, arrayTypeDef, "xs:sequence");
 
                     if (schema.valueSchema().type() == Type.STRUCT) {
                         for (final Field field : schema.valueSchema().fields()) {
                             processSchema(doc, arraySeq, field.name(), field.schema());
                         }
-                    }
-                    else if (schema.valueSchema().type() == Type.ARRAY) {
+                    } else if (schema.valueSchema().type() == Type.ARRAY) {
                         processSchema(doc, arraySeq, "entry", schema.valueSchema());
-                    }
-                    else if (schema.valueSchema().type() == Type.MAP) {
+                    } else if (schema.valueSchema().type() == Type.MAP) {
```

Comment:
```suggestion
                    } 
                    else if (schema.valueSchema().type() == Type.MAP) {
```

Unrelated to this specific PR